setup.cfg

```diff
@@ -0,0 +1,57 @@
+[metadata]
+name = aiida-testing
+author = Dominik Gresch
+author_email = greschd@gmx.ch
+version = 0.1.0.dev1
```

Comment:
We should use `attr: aiida_testing.__version__` instead of hard-coding the version. Then we can also get rid of the `validate_version_consistency` pre-commit check.